src/game/behaviors/castle_cannon_grate.inc.c

```diff
@@ -1,7 +1,7 @@
 // castle_cannon_grate.inc.c
 
 void bhv_castle_cannon_grate_init(void) {
-    if (save_file_get_total_star_count(gCurrSaveFileNum - 1, COURSE_MIN - 1, COURSE_MAX - 1) >= 120) {
+    if (save_file_get_total_star_count((gCurrSaveFileNum - 1), COURSE_NUM_TO_INDEX(COURSE_MIN), COURSE_NUM_TO_INDEX(COURSE_MAX)) >= 120) {
```

Comment:
this line is super long now and slower to read, i think `-1` is obvious that its an index. Alternatively, instead of a function macro i think just `COURSE_MIN_INDEX` and `COURSE_MAX_INDEX` is a lot more legible in this case

Alternatively Alternatively Alternatively, make a macro func like `cur_save_get_total_star_count()` which just calls `save_file_get_total_star_count` with all of those parameters